packages/cli-config/src/loadConfig.ts

```diff
@@ -89,6 +92,24 @@ function loadConfig(projectRoot: string = findProjectRoot()): Config {
     },
   };
 
+  // Try our best to figure out what version of React Native we're running. This is
+  // currently being used to get our deeplinks working, so it's only worried with
+  // the major and minor version.
+  try {
+    const {version} = JSON.parse(
+      fs
+        .readFileSync(path.join(initialConfig.reactNativePath, 'package.json'))
+        .toString(),
+    );
+    const out = semver.parse(version);
+    if (out) {
+      // Retain only these version, since they correspond with our documentation.
+      initialConfig.reactNativeVersion = `${out.major}.${out.minor}`;
+    }
+  } catch (_) {
+    // We don't seem to be in a well formed project, give up.
+  }
+
```

Comment:
This is a bit cheeky admittedly, I'm wondering if this is hiding somewhere (or there's a helper method that I should be using).

Reply:
Thanks @adamTrz  for the pointer.  I've added [another commit](ab21873c067f72904c98c0e71e5311aeb0e723dc), since `releaseChecker` doesn't quite do what I wanted here.  In that commit I've refactored that module to provide 3 things:

1. what is the current version in this project → but as a parsed semver object
2. what is the the next stable upgrade from the current version → effectively `releaseChecker` original, excluding the built in logging.
3. log the next available stable upgrade from the current version → `releaseChecker` as it was being used.

I've also cleaned up small things like consistent return types.

## TL;DR:

**I'm happy to submit [this change](ab21873c067f72904c98c0e71e5311aeb0e723dc) as another PR if the original one goes in (so I'd revert that commit on this branch).  This PR is already getting quite big.  Does that allays your original concern?**